src/shared/bodies/Wall.js

```diff
@@ -1,5 +1,6 @@
-import { Bodies, World } from 'matter-js'
+import { World } from 'matter-js'
```

Comment:
Need to import Bodies here until we have a parent class

Reply:
Ok, imported Bodies back into Wall.js.